services/whatsapp.ts

```diff
@@ -1,4 +1,4 @@
-import axios from "axios";
+import axios, { AxiosError } from "axios";
```

Comment:
while we are new, we want to use axios or the smaller native Fetch 😁 ?

Reply:
interesting, lets do it on new issue